### PR TITLE
Feature/product sku names

### PR DESCRIPTION
**Pull request by wender**
#### What problem is this solving?
Provide a list of the selected variations for the current product, also provide a source dependency to the `product-summary` 

#### How to test it?
Add the `product-sku-names` block to any block below `store.product` (Product template). For example:

```json
  "store.product": {
    "children": [
      "flex-layout.row#product",
    ]
  },
  "flex-layout.row#product": {
    "children": [
      "product-sku-names"
    ]
  },
```

[Check it working here](https://wender--alssports.myvtex.com/nixon-regulus-watch/p)

#### Screenshots or example usage:
image

### CODE REVIEW

**Review by klzns — CHANGES_REQUESTED** [review]
Did some changes here #866 let me know if it makes sense

File: react/ProductSKUNames.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { useCssHandles } from 'vtex.css-handles'
+import useProduct from 'vtex.product-context/useProduct'
+
+const CSS_HANDLES = [
+  'productSKUNamesContainer',
+  'skuLineClass',
+  'skuNameClass',
+  'skuValueClass',
```
Comment (klzns) [inline react/ProductSKUNames.tsx:9]: I think the name of this file and their CSS handles should be related to `ProductSKUAttributes` instead os skunames

**Review by klzns — CHANGES_REQUESTED** [review]

File: react/ProductSKUAttributes.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import ProductSKUAttributes from './components/ProductSKUAttributes/Wrapper'
```
Comment (klzns) [inline react/ProductSKUAttributes.js:1]: Remove file

File: react/components/ProductSKUAttributes/Wrapper.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import React from 'react'
```
Comment (klzns) [inline react/components/ProductSKUAttributes/Wrapper.js:1]: Remove file

**Review by klzns — APPROVED** [review]

File: docs/ProductSKUAttributes.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+📢 Use this project, [contribute](https://github.com/vtex-apps/store-components) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion). 
+
+# Product SKU Attributes
+
+The `ProductSKUAttributes` is a block responsible for **displaying a list of the currrent SKU variations** for the product.
+
```
Comment (guerreirobeatriz) [inline docs/ProductSKUAttributes.md:6]: Can you add here the image used to describe the PR?

File: docs/ProductSKUAttributes.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+## Configuration
+
+1. Import the `vtex.store-components` app to your theme's dependencies in the `manifest.json`, for example:
```
Comment (guerreirobeatriz) [inline docs/ProductSKUAttributes.md:9]:
```suggestion
1. Add the `vtex.store-components` app to your theme's dependencies in the `manifest.json` file:
```

File: docs/ProductSKUAttributes.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+```json
+  "dependencies: {
+    "vtex.store-components": "3.x"
+  }
+```
```
Comment (guerreirobeatriz) [inline docs/ProductSKUAttributes.md:15]:
```suggestion
```diff
  "dependencies: {
+   "vtex.store-components": "3.x"
  }
```
```

File: docs/ProductSKUAttributes.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+2. Add the `product-sku-attributes` block to any block below `store.product` (Product template). For example:
```
Comment (guerreirobeatriz) [inline docs/ProductSKUAttributes.md:17]:
```suggestion
2. Add the `product-sku-attributes` block to any block in the `store.product` template (Product Details Page template). For example:
```

File: docs/ProductSKUAttributes.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+      "flex-layout.row#product",
+    ]
+  },
+  "flex-layout.row#product": {
+    "children": [
+      "product-sku-attributes"
+    ]
+  },
+```
+
```
Comment (guerreirobeatriz) [inline docs/ProductSKUAttributes.md:31]: Hey, Wender! I have a question about this step. I'm not sure if I understood it correctly, but the idea here is to declare the block anywhere? And then it is going to be automatically rendered in the "right" place on the UI?

File: docs/ProductSKUAttributes.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+## Customization
+
+In order to apply CSS customizations in this and other blocks, follow the instructions given in the recipe on [Using CSS Handles for store customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization).
+
+| CSS Handles |
+| --- |
+| `ProductSKUAttributesContainer` |
+| `attributeValue` |
+| `attributeName` |
+| `attributeLine` |
```
Comment (guerreirobeatriz) [inline docs/ProductSKUAttributes.md:41]:
```suggestion
| CSS Handles |
| --- |
| `attributeLine` |
| `attributeName` |
| `attributeValue` |
| `ProductSKUAttributesContainer` |
```

**Review by guerreirobeatriz — CHANGES_REQUESTED** [review]

**Review by guerreirobeatriz — APPROVED** [review]

**Pull request merged**